cmd/inspector/main.go

```diff
@@ -636,29 +636,75 @@ func csvOutput() (*os.File, error) {
 	return os.Create(CSVPath)
 }
 
-func printSegmentHealthTable(w *csv.Writer, redundancy eestream.RedundancyStrategy, segments []*pb.SegmentHealth) error {
+func printSegmentHealthAndNodeTables(w *csv.Writer, redundancy eestream.RedundancyStrategy, segments []*pb.SegmentHealth) error {
 	segmentTableHeader := []string{
-		"Segment Index", "Online Nodes", "Offline Nodes",
+		"Segment Index", "Healthy Nodes", "Unhealthy Nodes", "Offline Nodes",
```

Comment:
The concept of "unreliable" nodes goes away when disqualification is done... we probably don't want to carry that through to "Healthy"/"Unhealthy" nodes.  Maybe this should be health / offline / disqualified?  Or leave it the old way?

Reply:
Four our purposes, unhealthy means online + disqualified